src/vs/editor/contrib/find/browser/find.ts

```diff
@@ -27,9 +28,10 @@ export class FindController extends CommonFindController implements IFindControl
 		@IContextKeyService contextKeyService: IContextKeyService,
 		@IKeybindingService keybindingService: IKeybindingService,
 		@IThemeService themeService: IThemeService,
-		@IStorageService storageService: IStorageService
+		@IStorageService storageService: IStorageService,
+		@IClipboardService clipBoardService: IClipboardService
```

Comment:
I recommend using `@optional(IClipboardService) clipBoardService: IClipboardService`
like it is used [here](https://github.com/Microsoft/vscode/blob/4b3f1820f358c47c2f4e0c34b65b2c375fa5b783/src/vs/editor/contrib/rename/browser/rename.ts#L106). This will help with the standalone editor (the monaco editor).

`optional` is defined in `vs/platform/instantiation/common/instantiation`

Then, the code should assume that this service could be undefined/null

> nitpick: perhaps name it `clipboardService` instead of `IClipboardService` to keep in spirit with the service name.